qemu/tests/cfg/netperf_udp_perf.cfg

```diff
@@ -0,0 +1,50 @@
+- netperf_udp_perf: install setup image_copy unattended_install.cdrom
+    #only linux
+    ver_cmd = rpm -qa qemu-kvm-rhev && rpm -qa qemu-kvm
+    mem_fixed = 4096
+    type = netperf_udp_perf
+    kill_vm = yes
+    image_snapshot = yes
+    setup_ksm = no
+    take_regular_screendumps = no
+    store_vm_register = no
+    nics += ' nic2'
+    netdst_nic1 = private
+    netdst_nic2 = switch
+    nettype = bridge
+    netdst = switch
+    nic_model_nic1 = virtio
+    vhost_nic1 = vhost=on
+    nic_model_nic2 = e1000
+    vhost_nic2 =
+    nic_mode = tap
+    numa_node = -1
+    netperf_with_numa = yes
+    burst_time = "1"
+    numbers_per_burst="10 50 100 150 180"
+    ver_cmd = rpm -q qemu-kvm
+    l = 20
+    netperf_version = 2.7.1
+    netperf_pkg = netperf/netperf-2.7.1.tar.bz2
+    setup_cmd = "cd /tmp && rm -rf netperf-2.7.1 && tar xvfj netperf-2.7.1.tar.bz2 && cd netperf-2.7.1 && sh autogen.sh && ./configure --enable-burst --enable-demo=yes --enable-intervals=yes && make"
+    log_hostinfo_script = scripts/rh_perf_log_hostinfo_script.sh
+    host_tuned_profile = "tuned-adm profile virtual-host"
+    client_tuned_profile = "tuned-adm profile virtual-host"
+    server_tuned_profile = "tuned-adm profile virtual-guest"
+    env_setup_cmd = "systemctl stop firewalld.service || service iptables stop;"
+    env_setup_cmd += " echo 2 > /proc/sys/net/ipv4/conf/all/arp_ignore;"
+    env_setup_cmd += " echo 0 > /sys/kernel/mm/ksm/run;"
+    env_setup_cmd += " echo 0 > /proc/sys/kernel/watchdog;"
+    env_setup_cmd += " echo 0 > /proc/sys/kernel/nmi_watchdog;"
+    env_setup_cmd += " setenforce 1"
+    #client configuration
+    username_client = root
+    shell_client_client = ssh
+    shell_port_client = 22
+    shell_prompt_client =  \[root@.{0,50}][\#\$]
+    #host configuration
+    shell_port_host = 22
+    username_host = root
+    os_type_client = linux
+    os_type_host = linux
+    shell_prompt_host =  \[root@.{0,50}][\#\$]
```

Comment:
Better keep this parameters here to let people know they need to specify them when run this test.
```
client_pub_ip = 
client_physical_nic = 
username_client = 
password_client = 
...
```

Reply:
I have added it  and uncomment it in cfg file